db/migrate/20190205142619_add_content_constraints.rb

```diff
@@ -0,0 +1,37 @@
+class AddContentConstraints < ActiveRecord::Migration[5.2]
+  def up
+    Katello::ProductContent.where(:content_id => nil).delete_all
+    Katello::ProductContent.where(:product_id => nil).delete_all
+
+    change_column :katello_product_contents, :content_id, :integer, :null => false
+    change_column :katello_product_contents, :product_id, :integer, :null => false
+
+    duplicates =  ActiveRecord::Base.connection.execute(
+        'select  "katello_contents"."cp_content_id", "katello_contents"."organization_id"
+          FROM "katello_contents"
+          GROUP BY "katello_contents"."cp_content_id", "katello_contents"."organization_id"
+          HAVING (count(*) > 1)')
```

Comment:
I tried various active record ways of getting this data, but none seemed to work, open to suggestions

Reply:
`Katello::Content.having('count(*) > 1').group(:cp_content_id, :organization_id).select(:cp_content_id, :organization_id)` ?

Reply:
nice, i swear i tried that (but probably tried very similar things), that seems to work.

Reply:
pushed this change, thanks!